src/Cms/PageRules.php

```diff
@@ -293,6 +297,25 @@ public static function update(Page $page, array $content = []): bool
             ]);
         }
 
+        self::slugMaxlength($page);
+
         return true;
     }
+
+    protected static function slugMaxlength(Page $page, string $slug = null): void
+    {
+        if ($slugsMaxlength = option('slugs.maxlength')) {
```

Comment:
I think there should be a default value, e.g. 150 characters. This default value should be used if the configured option is not set or if it's not already an integer.

So the `if` is not required, the check should always be made, either with the configured value or with the default.